styles.css

```diff
@@ -72,7 +82,7 @@ header {
   width: 25px;
   border-top-right-radius: 35px;
   box-shadow: 0 -24px 0 0 #F6EC5F;
-}
+} */
 
```

Comment:
For the most part, we should probably delete unused code after a better solution is made.